baldrick/__init__.py

```diff
@@ -31,6 +32,13 @@ def create_app(name, register_blueprints=True):
 
     app.wsgi_app = ProxyFix(app.wsgi_app)
 
+    # Check if there is a global configuration
+    global_toml = os.path.join(os.path.expanduser('~'), 'pyproject.toml')
```

Comment:
why `~` and not just the current directory?

Reply:
Hmm yeah maybe that makes more sense.